script_test.go

```diff
@@ -1850,6 +1850,22 @@ func TestReadReturnsErrorGivenReadErrorOnPipe(t *testing.T) {
 	}
 }
 
+func TestWaitForErrorOnPipe(t *testing.T) {
```

Comment:
If you try `gotestdox` here, as discussed on the other PR, you'll see why this test name doesn't quite fit with the others:

```
 ✔ Error returns error set by previous pipe stage (0.00s)
...

 ✔ Wait reads pipe source to completion (0.00s)
...
 ✔ Wait for error on pipe (0.00s)
 ✔ Wait for no error on pipe (0.00s)
```

See what I mean? It's not a sentence describing the effect that `Wait` _has_. (This is why I find `gotestdox` so useful as a thinking tool: it forces me to describe the behaviour of the thing I'm implementing.)

Reply:
Thanks for the great suggestion @bitfield! changed the test names to conform to your suggestion. Let me know what you think.